src/components/common/Job/jobTitle.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { WithStyles, withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import LocationOnOutlinedIcon from '@material-ui/icons/LocationOnOutlined';
+import styles from './jobStyles';
+import { Advertisement } from '../../../types/Advertisements';
+
+interface JobTitleData {
+    ad: Advertisement;
+    showLocation: boolean;
+}
+
+type JobTitleProps = JobTitleData & WithStyles<typeof styles>;
+
+const JobTitle: React.FC<JobTitleProps> = (props): JSX.Element => {
```

Comment:
💯 Quite brilliant solution!!!! I like the splitting into smaller files that are used more than once 🥳